### ISSUE TITLE
capistrano 3.7.0 warning

### ISSUE THREAD

**Issue by dreyks** [seuros/capistrano-sidekiq#160]
After upgrading to capistrano 3.7.0 the following warning pops up on deploy

Skipping task 'sidekiq:stop'.
Capistrano tasks may only be invoked once. Since task `sidekiq:stop' was previously invoked, invoke("sidekiq:stop") at <path_to_gems>/capistrano-sidekiq-0.5.4/lib/capistrano/tasks/sidekiq.rake:172 will be skipped.
If you really meant to run this task again, first call Rake::Task["sidekiq:stop"].reenable
THIS BEHAVIOR MAY CHANGE IN A FUTURE VERSION OF CAPISTRANO. Please join the conversation here if this affects you.
https://github.com/capistrano/capistrano/issues/1686

**Comment by gaogao1030** [comment]
same problem for me

**Comment by attenzione** [comment]
everyone, who uses new 3.7 version will get this warning

**Comment by musashinm** [comment]
I'm with Cap 3.6 and I got the same problem.

**Comment by moiristo** [comment]
+1

**Comment by Qasem-h** [comment]
+1

**Comment by williamn** [comment]
Any idea how to handle this?

**Comment by lloydwatkin** [comment]
I was seeing the same on 3.6.1 (and annoyingly this left me with several sidekiq jobs running per host). However #164 I no longer see the warning and I'm back to single instances of sidekiq jobs \o/

**Comment by lloydwatkin** [comment]
It would be good to get a new gem released though :)

**Comment by kyledecot** [comment]
+1

**Comment by IvoGuerreiro** [comment]
+1

**Comment by jrtowell** [comment]
+1

**Comment by maduhaime** [comment]
+1

**Comment by absolu7** [comment]
+1

**Comment by danilogcarolino** [comment]
+1

**Comment by marlosirapuan** [comment]
+1

**Comment by mdesantis** [comment]
+1

**Comment by ivanovaleksey** [comment]
Hi guys!
I also have this problem with Capistrano 3.6.1.
I use master branch in my project like
```
gem 'capistrano-sidekiq', github: 'seuros/capistrano-sidekiq'
```
and it seems to be working fine.

How about to push new gem version?
It is more than a year now since [last release](https://rubygems.org/gems/capistrano-sidekiq/versions).

**Comment by ignisf** [comment]
Hey, @seuros, thanks for https://github.com/seuros/capistrano-sidekiq/commit/339c42e24f42d8d72173daae832b23504930f941 and releasing the new version. Just writing a comment here to ping everybody subscribed to this issue. :)

**Comment by gregawoods** [comment]
I also started seeing this warning today. Using capistrano 3.10.1 and capistrano-sidekiq 1.0.1. If it makes any difference, I am using `capistrano/sidekiq/monit` as well.

```
00:35 sidekiq:start
      01 /usr/local/rvm/bin/rvm 2.4.2 do bundle exec sidekiq --index 0 --pidfile /var/deploy/example/shared/tmp/pids/sidekiq-0.pid --environment production --logfile /var/deploy/example
    ✔ 01 deploy@example.net 0.817s
Skipping task `sidekiq:restart'.
Capistrano tasks may only be invoked once. Since task `sidekiq:restart' was previously invoked, invoke("sidekiq:restart") at /Users/gregwoods/.rvm/gems/ruby-2.4.2/gems/capistrano-sidekiq-1.0.1/lib/capistrano/tasks/sidekiq.rake:34 will be skipped.
If you really meant to run this task again, use invoke!("sidekiq:restart")
THIS BEHAVIOR MAY CHANGE IN A FUTURE VERSION OF CAPISTRANO. Please join the conversation here if this affects you.
https://github.com/capistrano/capistrano/issues/1686
00:36 sidekiq:monit:monitor
      01 sudo /usr/bin/monit monitor sidekiq_example_production_0
    ✔ 01 deploy@example.net 0.130s
```

**Comment by scmx** [comment]
Same here `capistrano (3.10.1)` and `capistrano-sidekiq (1.0.1)`
```
$ cap --trace example deploy
...
** Invoke deploy:restart_sidekiq (first_time)
** Execute deploy:restart_sidekiq
** Invoke sidekiq:restart (first_time)
** Execute sidekiq:restart
** Invoke sidekiq:stop (first_time)
** Execute sidekiq:stop
** Invoke sidekiq:start (first_time)
** Execute sidekiq:start
00:22 sidekiq:start
      01 RBENV_ROOT=$HOME/.rbenv RBENV_VERSION=2.4.3 $HOME/.rbenv/bin/rbenv exec bundle exec sidekiq --index 0 --pidfile /example/…
    ✔ 01 example@example 2.282s
Skipping task `sidekiq:restart'.
Capistrano tasks may only be invoked once. Since task `sidekiq:restart' was previously invoked, invoke("sidekiq:restart") at capistrano-sidekiq-1.0.1/lib/capistrano/tasks/sidekiq.rake:34 will be skipped.
If you really meant to run this task again, use invoke!("sidekiq:restart")
THIS BEHAVIOR MAY CHANGE IN A FUTURE VERSION OF CAPISTRANO. Please join the conversation here if this affects you.
https://github.com/capistrano/capistrano/issues/1686
** Invoke sidekiq:restart
** Invoke deploy:published (first_time)
** Execute deploy:published
** Invoke sidekiq:start
** Invoke sidekiq:monit:monitor (first_time)
** Execute sidekiq:monit:monitor
...
```

**Comment by seuros** [comment]
fixed in 2a9e46f and 1.0.2
